src/recordlinker/linking/link.py

```diff
@@ -29,53 +29,6 @@
 
     TRACER = MockTracer()
 
-
-# TODO: This is a FHIR specific function, should be moved to a FHIR module
-def fhir_record_to_pii_record(fhir_record: dict) -> schemas.PIIRecord:
-    """
-    Parse the FHIR record into a PIIRecord object
-    """
-    val = {
-        "external_id": fhir_record.get("id"),
-        "name": fhir_record.get("name", []),
-        "birthDate": fhir_record.get("birthDate"),
-        "sex": fhir_record.get("gender"),
-        "address": fhir_record.get("address", []),
-        "mrn": None,
-        "ssn": None,
-        "race": None,
-        "gender": None,
-        "telecom": fhir_record.get("telecom", []),
-    }
-    for identifier in fhir_record.get("identifier", []):
-        for coding in identifier.get("type", {}).get("coding", []):
-            if coding.get("code") == "MR":
-                val["mrn"] = identifier.get("value")
-            elif coding.get("code") == "SS":
-                val["ssn"] = identifier.get("value")
-    for address in val["address"]:
-        address["county"] = address.get("district", "")
-        for extension in address.get("extension", []):
-            if extension.get("url") == "http://hl7.org/fhir/StructureDefinition/geolocation":
-                for coord in extension.get("extension", []):
-                    if coord.get("url") == "latitude":
-                        address["latitude"] = coord.get("valueDecimal")
-                    elif coord.get("url") == "longitude":
-                        address["longitude"] = coord.get("valueDecimal")
-    for extension in fhir_record.get("extension", []):
-        if extension.get("url") == "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race":
-            for ext in extension.get("extension", []):
-                if ext.get("url") == "ombCategory":
-                    val["race"] = ext.get("valueCoding", {}).get("display")
-        if extension.get("url") == "http://hl7.org/fhir/StructureDefinition/individual-genderIdentity":
-            for ext in extension.get("extension", []):
-                if ext.get("url") == "value":
-                    for coding in ext.get("valueCodeableConcept", {}).get("coding", []):
-                        val["gender"] = coding.get("display")
-
-    return schemas.PIIRecord(**val)
-
-
 # TODO: This is a FHIR specific function, should be moved to a FHIR module
 def add_person_resource(
```

Comment:
@cbrinson-rise8 I'm thinking this method should move as well since its specific to creating a FHIR bundle.  What do you think?

Reply:
Yeah, I can move that one as well. The file I created is under `recordlinker.parsers.fhir.py` I know this function isn't specifically parsing but more of a util. Thoughts?

Reply:
Maybe we can just call this the `hl7` module?  All the planned formats are HL7 specs

Reply:
Oh sorry, I was vague with my response.  I was thinking `recordlinker.hl7.fhir`, `recordlinker.hl7.ecr`, `recordlinker.hl7.elr`.  So all those hl7 specs that we want to parse (and in FHIR's case also serialize), are placed into the `recordlinker.hl7` package.

Reply:
That way we can put both functions for parsing FHIR and encoding it (aka making a bundle) directly in the `recordlinker.hl7.fhir` module, as its no longer specific to just the former.

Reply:
oh gotcha. yeah I'll make that change real quick